extensions/azdata/package.nls.json

```diff
@@ -2,5 +2,14 @@
 	"azdata.displayName": "Azure Data CLI",
 	"azdata.description": "Support for Azure Data CLI.",
 	"azdata.config.title": "Azure Data CLI Configuration",
-	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands"
+	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands",
+
+	"azdata.acceptEula.command.name": "Accept Eula",
+	"azdata.install.command.name": "Install",
+	"azdata.upgrade.command.name": "Upgrade",
+	"azdata.category": "Azdata",
```

Comment:
`Azure Data CLI`

Reply:
really? This will make the command look like the following:

Azure Data CLI: Accept Azure Data CLI Eula

So I will change the category to 'Azure Data CLI' and remove the 'Azure Data CLI' from the command specific strings so that the ultimate commands will show up as follows:

Azure Data CLI: Accept Eula
Azure Data CLI: Install
Azure Data CLI: Check for Update

Reply:
Yup, sounds good to me.

Reply:
Done